src/fileStorage.ts

```diff
@@ -0,0 +1,27 @@
+import * as Minio from 'minio'
+
+import environment from './environment'
+
+const requiredBuckets = ['graders', 'submissions']
+
+const minioConfiguration: Minio.ClientOptions = {
+  endPoint: environment.minioHost,
+  port: environment.minioPort,
+  useSSL: false,
+  accessKey: environment.minioUsername,
+  secretKey: environment.minioPassword,
+}
+
+export const minioClient = new Minio.Client(minioConfiguration)
+
+export async function initializeMinio() {
```

Comment:
I'd say that's debatable, but I'll change the code because I don't feel like having the full debate rn. My counterpoint is that the continue is easy to miss leaving the reader thinking that make bucket is called unconditionally. When it's nested in a conditional it's clear that it runs conditionally